src/storage-client/src/controller.rs

```diff
@@ -739,40 +708,68 @@ pub trait StorageController: Debug {
     ) -> Result<(), StorageError<Self::Timestamp>>;
 }
 
+/// A wrapper struct that presents the adapter token to a format that is understandable by persist
+/// and also allows us to differentiate between a token being present versus being set for the
+/// first time.
+// TODO(aljoscha): Make this crate-public again once the remap operator doesn't
+// hold a critical handle anymore.
+#[derive(PartialEq, Clone, Debug)]
+pub struct PersistEpoch(pub Option<NonZeroI64>);
```

Comment:
Looks like the code compiles with `pub(crate)` as well. Is the TODO comment outdated or does it look into the future?

Reply:
I think you're right!

Reply:
Ah nope, it doesn't compile because this is now in the client crate but the storage controller in the controller crate also uses it.

Reply:
Oh, you are right. I swear I tested this, but looks like I messed up 🙈